src/Commands/SSHKey/AddCommand.php

```diff
@@ -1,7 +1,7 @@
 <?php
 /**
  * @file
- * Contains Pantheon\Terminus\Commands\Auth\SSHKey\AddCommand
+ * Contains Pantheon\Terminus\Commands\SSHKey\AddCommand
```

Comment:
@TeslaDethray: My PHPStorm adds these little PHPDoc gremlins to every class file but you don't use them. Should I tell it to stop? I never really saw the point in them.